exercises/practice/list-ops/.meta/example.s

```diff
@@ -0,0 +1,182 @@
+.text
+.globl append
+.globl filter
+.globl map
+.globl foldl
+.globl foldr
+.globl reverse
+
+
+/* extern size_t append(int64_t* buffer, const int64_t* list1, size_t list1_count, const int64_t* list2, size_t list2_count); */
```

Comment:
To be consistent with `list_ops_test.c` (but really nit picking here ...):
```suggestion
/* extern size_t append(int64_t *buffer, const int64_t *list1, size_t list1_count, const int64_t *list2, size_t list2_count); */
```
Also applies to other C function signatures in this file.